pkg/diagnose/diagnosis/loader.go

```diff
@@ -0,0 +1,27 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+package diagnosis
```

Comment:
As the catalog will be imported by every package providing a diag (to be able to register it), it's better to move it to its own package, so avoid importing the whole package everytime (although it's pretty lightweight by now)

Reply:
It's already in its own `diagnosis` package, or do you mean something else?

Reply:
my bad, got confused between the `diagnose` and `diagnosis` package names